integrations/tensorflow/e2e/README.md

```diff
@@ -1,7 +1,7 @@
 # TensorFlow e2e tests
 
-This is a collection of e2e tests that save a TensorFlow model, compile it with
-IREE, run it on multiple backends and crosscheck the results.
+This is a collection of e2e tests that compile a TensorFlow model with IREE (and
+potentially TFLite), run it on multiple backends, and crosscheck the results.
```

Comment:
maybe for completeness, mention that we run on TF as a reference?

Reply:
Already mentioned in the section about tests! This section just covers the functionality of `IreeCompiledModule`.